test/e2e/operator_test.go

```diff
@@ -364,6 +380,59 @@ func customResolver(nameserver string) *net.Resolver {
 	}
 }
 
+func ensureOperandNamespace() error {
+	err := kubeClient.Create(context.TODO(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "external-dns"}})
+	return err
+}
+
+func ensureOperandRole() error {
+	rules := []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{""},
+			Resources: []string{"secrets", "serviceaccounts", "namespaces"},
+			Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
+		},
+		{
+			APIGroups: []string{"apps"},
+			Resources: []string{"deployments"},
+			Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
+		},
+	}
+
+	role := rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "external-dns-operator",
+			Namespace: "external-dns",
+		},
+		Rules: rules,
+	}
+	err := kubeClient.Create(context.TODO(), &role)
+	return err
+}
+
+func ensureOperandRoleBinding() error {
+	rb := rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "external-dns-operator",
+			Namespace: "external-dns",
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: rbacv1.GroupName,
+			Kind:     "Role",
+			Name:     "external-dns-operator",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      rbacv1.ServiceAccountKind,
+				Name:      "external-dns-operator",
+				Namespace: "external-dns-operator",
```

Comment:
Would you mind making those constants too?

Reply:
Done. Thx